BillingApp/BillingInfo/templates/test_show_all.html

```diff
@@ -2,41 +2,68 @@
 <head>
     <meta charset="UTF-8">
     <title>TEST</title>
+    <style>
```

Comment:
we could extract this to a css file in `static/..`, see [this](https://docs.djangoproject.com/en/3.0/intro/tutorial06/)